### DISCUSSION TITLE
Handling Services with Missing Rates in MRF Posting

### DISCUSSION THREAD

**Discussion by siddesai1005** [CMSgov/hospital-price-transparency#80] [Q&A] [answered]
Should services listed in the description column without established rates or values for standard gross charges, discounted cash, negotiated dollar/%/algorithm be retained in the MRF while posting, or deleted while posting files on the website? The reason could be either those services are not performed at the facility or no established rate. Currently, it shows an error when uploaded on the price transparency tool validator, indicating that it is not encoded with one of the specified categories: "Gross Charge," "Discounted Cash Price," "Payer-Specific Negotiated Charge: Dollar Amount," "Payer-Specific Negotiated Charge: Percentage," or "Payer-Specific Negotiated Charge."

**Comment by CarlsonErin** [comment, marked as answer]
If you have not established any of the five standard charges for a hospital item or service, you are not required to include that item or service in your machine-readable file.  Please only include hospital items and services for which you have established a standard charge.
Below are additional reminders to avoid common errors in MRFs:
•	Do not insert a value or any type of indicators (e.g., “N/A” or “0”) if the hospital does not have applicable data to encode. If you would like to include an explanation for the blanks, you may do so using Additional Generic Notes or Additional Payer-Specific Notes.
•	Encode valid values as instructed below. Values encoded incorrectly will generate a deficiency.
o	For example, if the valid value is ‘numeric’ (such as for Payer-Specific Negotiated Charge: Dollar Amount), inserting anything other than a number (such as inserting a dollar sign with a number) will generate a deficiency. Similarly, if the valid value is ‘enum’ (such as for Code Type), inserting anything other than the values indicated (such as inserting ‘other’) will generate a deficiency.
o	All "Numeric" data elements must be positive numbers. Entering a negative number or "0" will generate a deficiency.
Please review the [conditional requirements](https://github.com/CMSgov/hospital-price-transparency/tree/master/documentation/CSV#conditional-requirements) as well.

**Comment by kathyreep** [comment]
In response to the information below, many pharmacy items do not carry a charge in the chargemaster. It is established at the time of service based on current cost to the provider. Is the charge established at the time of service considered the standard charge as it could vary from week to week…

Kathy A. Reep
Consulting & Business Development | PYA, P.C.

D (727) 450-6948 | C (407) 492-7519
100 N. Tampa Street, Suite 1850
Tampa, FL 33602

**Reply by RidenourK** [reply]
@kathyreep. Thank you for your question. 

Please see the responses in post #82.
